scripts/ci/pylint_todo.txt

```diff
@@ -201,6 +201,7 @@
 ./airflow/kubernetes/kubernetes_request_factory/kubernetes_request_factory.py
 ./airflow/kubernetes/kubernetes_request_factory/pod_request_factory.py
 ./airflow/kubernetes/kube_client.py
+./airflow/kubernetes/refresh_config.py
```

Comment:
Please don't add new files in to pylint todo, instead fix whatever pylint is complaining about :)

Reply:
+1

Reply:
I initially tried fixing all the pylint errors, but the code turned to be much different than the original version. So I added it here to keep it similar to original k8s code to it's easier to compare with upstream code for further changes if needed. Turns out k8s python client doesn't pass our pylint rule ;)

With that, do you still want me to fix all the pylint error? If so, I can throw in my original patches.

Reply:
What is the pylint error in this new file? It should be possible to fix this file without changing anything else.

Reply:
I found a better way to handle this, with the new approach, we won't have to add new entry to pylint_todo.txt and we get to preserve the source code structure of upstream k8s client for easier diffing going forward.